service/scaladsl/kafka/server/src/test/scala/com/lightbend/lagom/scaladsl/kafka/broker/ScaladslKafkaApiSpec.scala

```diff
@@ -70,7 +68,7 @@ class ScaladslKafkaApiSpec extends WordSpecLike with Matchers with BeforeAndAfte
     super.afterAll()
   }
 
-  implicit val patience = PatienceConfig(30.seconds, 150.millis)
+  implicit val patience = PatienceConfig(35.seconds, 300.millis)
```

Comment:
this change is for `self-heal if failure occurs while running the publishing stream` (see PRs description)